BOJ/1043_정준우.cpp

```diff
@@ -0,0 +1,42 @@
+#include<iostream>
+#include<vector>
+using namespace std;
+
+vector<int> graph[101];
```

Comment:
```suggestion
vector<int> graph[51][2];
```
밑에 것들도 마찬가지이지만 1차원을 인덱50을 기준으로 나눠 쓰는것보다 2차원으로 나누는게 깔끔할거 같아요.

Reply:
2차원으로 하면 dfs 실행이 불가능해서 1차원으로 만들었습니당:)